python/lancedb/context.py

```diff
@@ -140,6 +140,14 @@ def text_col(self, text_col: str) -> Contextualizer:
     def to_df(self) -> pd.DataFrame:
         """Create the context windows and return a DataFrame."""
 
+        if self._window is None:
```

Comment:
I think we also want to validate `_textcol`. Could you add that?

Reply:
ya sure !